modules/cves-tab.adoc

```diff
@@ -12,11 +12,6 @@ The CVEs view organizes information into the following groups:
 * *Images by severity*: Groups images based on the severity level of the associated vulnerabilities.
 * *Top CVSS*: Displays the highest CVSS score for each CVE across images to highlight the vulnerabilities with the most severe impact.
 * *Top NVD CVSS*: Shows the highest severity scores from the National Vulnerability Database (NVD) to enable standardized impact assessments.
-+
-[NOTE]
-====
-You can see the *Top NVD CVSS* column only if you have enabled Scanner V4.
```

Comment:
do we still want to call out things which are only available in Scanner V4? This may help encourage older users to migrate to it. EPSS is another thing which we only offer with Scanner V4